### PR TITLE
[MLOB-1616] Add tracer version to auto-instrumentation table

### PR DESCRIPTION
**Pull request by sabrenner**
### What does this PR do? What is the motivation?
Adds a tracer (`ddtrace`) version column to our auto-instrumentation table, which is the version the LLMObs auto-instrumentation was introduced to the Python tracer.

### Merge instructions

- [x] Please merge after reviewing

### Additional notes
We are not including Gemini in this PR, we'll have another PR that documents our Gemini support, and part of that PR will include the tracer version that introduced the LLMObs integration.

### CODE REVIEW

**Review by lievan — APPROVED** [review]

File: content/en/llm_observability/setup/auto_instrumentation.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ further_reading:
...
-|-----------------------------------------|--------------------|
-| [OpenAI](#openai)                       | >= 0.26.5          |
-| [Langchain](#langchain)                 | >= 0.0.192         |
-| [AWS Bedrock](#aws-bedrock)             | >= 1.31.57         |
-| [Anthropic](#anthropic)                 | >= 0.28.0          |
+| Framework                               | Supported Versions | Tracer Version    |
+|-----------------------------------------|--------------------|-------------------|
+| [OpenAI](#openai)                       | >= 0.26.5          | >= 2.9.0          |
+| [Langchain](#langchain)                 | >= 0.0.192         | >= 2.9.0          |
+| [AWS Bedrock](#aws-bedrock)             | >= 1.31.57         | >= 2.9.0          |
```
Comment (hestonhoffman) [inline content/en/llm_observability/setup/auto_instrumentation.md:23]:
```suggestion
| [Amazon Bedrock](#amazon-bedrock)             | >= 1.31.57         | >= 2.9.0          |
```
Looks like Amazon calls this "Amazon Bedrock" (link)[https://aws.amazon.com/bedrock/?sec=aiapps&pos=2]. Would you mind changing it on line 85 as well?

**Review by hestonhoffman — COMMENTED** [review]
One small request to satisfy our style guide

File: content/en/llm_observability/setup/auto_instrumentation.md (hunk 1 above)
Comment (sabrenner) [reply content/en/llm_observability/setup/auto_instrumentation.md:23]: ah yes, can do! I was wondering what the fix to the check would be 😂

Comment (sabrenner) [reply content/en/llm_observability/setup/auto_instrumentation.md]: fixed, we also referenced `AWS` a couple of other (non-link) places on this page that I fixed as well

Comment (hestonhoffman) [reply content/en/llm_observability/setup/auto_instrumentation.md]: Yeah, usually it gives a recommendation 😅

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**